web_common/web_common/styles/main.scss

```diff
@@ -452,3 +452,9 @@ td a.fill-td {
   width: 100%;
   height: 100%;
 }
+//Progress bar css
+
+div .bar {
```

Comment:
I'm sorry. I should have been more clear. Are you sure we need this change? If so, can we add something that corresponds to the id of the div the plot is in (`plotly-timeseries`)? I think it's this:

```css
#plotly-timeseries .bar {
    display: inline-block;
    margin: 0;
}
```

Reply:
Oh Okay. I get what you mean. I am going to try it but I think that specific id is directly correlated with Plotly and the css it has attached to it, so we might not need any of the CSS specifically. But let me run it and make sure.

Reply:
That's what I thought. I don't think you need this at all.